app/src/main/java/org/wikipedia/page/leadimages/PageHeaderView.java

```diff
@@ -32,10 +33,13 @@
 public class PageHeaderView extends LinearLayoutOverWebView implements ObservableWebView.OnScrollChangeListener {
     @BindView(R.id.view_page_header_image) FaceAndColorDetectImageView image;
     @BindView(R.id.view_page_header_image_gradient) View gradientView;
+    @BindView(R.id.call_to_action_container) View callToActionContainer;
+    @BindView(R.id.call_to_action_text) TextView callToActionTextView;
     @Nullable private Callback callback;
 
     public interface Callback {
         void onImageClicked();
+        void onAddCaptionClicked();
```

Comment:
I like the fact that all of your other call-to-action fields and variables have purpose-agnostic names, like `callToActionTextView` and `setupCallToAction()`.  So then, shouldn't this callback function also have an agnostic name, such as `onCallToActionClicked()`?